Other/MergeShops/BloodWarLycanMerge.cs

```diff
@@ -0,0 +1,103 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/CoreAdvanced.cs
+//cs_include Scripts/Story/BloodMoon.cs
+using RBot;
+using RBot.Items;
+using RBot.Options;
+
+public class BloodWarLycanMerge
+{
+    public ScriptInterface Bot => ScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreFarms Farm = new();
+    public CoreStory Story = new();
```

Comment:
this public part isnt needed only the include for story.

Reply:
so i would change to what exactly? just remove the public part? this was generated automatically by template helper

Reply:
hmm if it generated that.. i guess 👍  I must just be blind nvm me 😞

Reply:
no, you are right core story is not used anywhere in the script but for some reason it is automatically added by the template. Do I remove it or not?